ripe/atlas/tools/renderers/probes.py

```diff
@@ -52,13 +54,13 @@ def _construct_probe_line(self):
 
         if not self.custom_format_flag:
             self.fields = ["id", "asn_v4", "asn_v6", "country_code", "status"]
-            self.probe_template = "{:<5}|{:<6}|{:<6}|{:^2}|{:<}|"
+            self.probe_template = "{:<5} {:<6} {:<6} {:^2} {:<}"
         else:
             for field in self.fields:
-                self.probe_template += "{:<}|"
+                self.probe_template += " {:<}"
 
         for fields in self.additional_fields:
-            self.probe_template += "{:<}|"
+            self.probe_template += " {:<}"
 
```

Comment:
Oh I'm sure it breaks everything, but I didn't want to bother tweaking the tests if you weren't cool with the changes.